src/pages/en/guides/content-collections.mdx

```diff
@@ -16,166 +16,196 @@ import TypeScriptSettingTabs from '~/components/tabs/TypeScriptSettingTabs.astro
+### Organizing with multiple collections 
 
-Content within a collection should share the same frontmatter shape and types. You can optionally enforce these types [by configuring a schema](/en/guides/content-collections/#defining-a-collection-schema).
+If two files represent two different kinds of content (e.g. a blog post and an author profile), they most likely belong in two different collections. This is an important concept to remember because important features like frontmatter validation and automatic TypeScript typings require that all entries share a similar frontmatter structure.
 
-To create a collection, add a new directory to `src/content/`. Then, add Markdown or MDX entries that share frontmatter properties. The following example shows two collections: `blog` and `newsletter`. 
+If you find yourself working with different types of content, you should create multiple collection to represent each type. You can create as many different collections in your project as you'd like.
 
 <FileTree>
 - src/content/
-  - **blog/** All blog posts have the same frontmatter properties
-    - columbia.md
-    - endeavour.md
-    - enterprise.md
-  - **newsletter/** All newsletters have the same frontmatter properties
+  - **newsletter/** 
     - week-1.md
     - week-2.md
     - week-3.md
+  - **authors/** split different content types into new collections
+    - grace-hopper.md
+    - alan-turing.md
+    - batman.md
 </FileTree>
 
-### Collections with nested directories
 
-Collections are **top-level folders** within `src/content/`. You cannot nest collections, but you may use nested directories within a collection to better organize a collection's content. All nested directories will share the same schema defined for the top-level collection.
+### Organizing with subdirectories
 
-For example, you can use this structure for internationalization:
+A content collection is always a top-level folder inside of the `src/content/` directory. You cannot nest one collection inside of another. However, you can use subdirectories to organize your content within a collection.
+
+For example, you can use the following directory structure to organize i18n translations within a single `docs` collection. When you go to query this collection, you'll be able to filter the result by language using the file path. 
 
 <FileTree>
 - src/content/
-  - docs/ docs schema applies to all nested directories
-    - en/
-    - es/
-    - ...
+  - docs/ this collection uses subdirectories to organize by language
+    - **en/**
+    - **es/**
+    - **de/**
 </FileTree>
 
-## Configuring collections
+## Defining Collections
 
-You can configure your content collections with an optional `src/content/config.ts` file (`.js` and `.mjs` extensions are also supported). 
+:::caution
```

Comment:
We know from exprience, it's a scary look in docs to have a caution right under a heading! Especially a "normal" heading like "Defining Collections." -> "Oh, let's see how to define my collection.... FLAMING CAUTION"

This caution should be moved to the bottom of the section, and I'd argue a caution is perhaps overkill for this and the text can stand on its own. The sentiment is fine, but at best this seems like a "note" or a "tip", which is at least a little more playful.

Reply:
no worries, I'll switch to note!